webpack.config.js

```diff
@@ -72,98 +32,79 @@ module.exports = merge(
         }
       },
       minimizer: [
-        // new OptimizeCSSAssetsPlugin({}),
-        new TerserPlugin(
-          {
-            terserOptions: {
-              module: false,
-              parse: {
-                // we want terser to parse ecma 8 code. However, we don't want it
-                // to apply any minfication steps that turns valid ecma 5 code
-                // into invalid ecma 5 code. This is why the 'compress' and 'output'
-                // sections only apply transformations that are ecma 5 safe
-                // https://github.com/facebook/create-react-app/pull/4234
-                ecma: 8
-              },
-              compress: {
-                ecma: 5,
-                warnings: false,
-                // Disabled because of an issue with Uglify breaking seemingly valid code:
-                // https://github.com/facebook/create-react-app/issues/2376
-                // Pending further investigation:
-                // https://github.com/mishoo/UglifyJS2/issues/2011
-                comparisons: false
-              },
+        new TerserPlugin({
+          terserOptions: {
+            module: false,
+            parse: {
+              // we want terser to parse ecma 8 code. However, we don't want it
+              // to apply any minfication steps that turns valid ecma 5 code
+              // into invalid ecma 5 code. This is why the 'compress' and 'output'
+              // sections only apply transformations that are ecma 5 safe
+              // https://github.com/facebook/create-react-app/pull/4234
+              ecma: 8
+            },
+            compress: {
+              ecma: 5,
+              warnings: false,
+              // Disabled because of an issue with Uglify breaking seemingly valid code:
+              // https://github.com/facebook/create-react-app/issues/2376
+              // Pending further investigation:
+              // https://github.com/mishoo/UglifyJS2/issues/2011
+              comparisons: false
+            },
+            keep_fnames: true,
```

Comment:
for compatibility reason keep functions names just like at the source files - don't try to shorten it with random names